crates/katana-core/src/sequencer.rs

```diff
@@ -112,7 +111,18 @@ impl KatanaSequencer {
         Ok((tx_hash, contract_address))
     }
 
-    pub async fn starknet_get_storage_at(
+    pub async fn get_class_hash_at(
```

Comment:
```suggestion
    pub async fn class_hash_at(
```

nit: rust convention is not to prefix getters with `get_`